src/storage/hnsw.hpp

```diff
@@ -34,12 +34,15 @@
 #include "dbtype_def.h"
 #include "thread_compat.hpp"
 #include "faiss/IndexHNSW.h"
+#include "faiss/IndexIDMap.h"
 
 BTID *xhnsw_add_index (THREAD_ENTRY *thread_p, BTID *btid, int dimension, int hnsw_M, int hnsw_efConstruction,
 		       enum faiss::MetricType metric_type);
 int xhnsw_delete_index (THREAD_ENTRY *thread_p, BTID *btid);
 int hnsw_print_index_info (BTID *btid);
-int hnsw_add_element (BTID *btid, DB_VALUE *key_dbvalue);
+int hnsw_add_element (BTID *btid, DB_VALUE *key_dbvalue, OID *oid);
```

Comment:
Key-Value 순서로 나열하는게 좋습니다.

```suggestion
int hnsw_add_element (BTID *btid, OID *oid, DB_VALUE *key_dbvalue);
```